package.json

```diff
@@ -12,7 +12,8 @@
     "build:docs:receipt": "jsdoc2md lib/receipt.js > Docs/receipt.md",
     "build:docs:monetary-amount": "jsdoc2md lib/monetary-amount.js > Docs/monetary-amount.md",
     "build:docs:utils": "jsdoc2md lib/utils.js > Docs/utils.md",
-    "test": "nyc mocha lib/**/*.spec.js --exit"
+    "test": "nyc mocha lib/**/*.spec.js --exit",
+    "test:dev": "nyc mocha lib/**/*.spec.js --exit --watch"
```

Comment:
Would `test:watch` be more accurate? `test:dev` implies to me that the tests are run in a different environment, or even a different test of tests.

Reply:
The reason that I chose `dev` instead of `watch` is because in my mind, while developing, I am in a special kind of _dev_ environment, that has different requirements compared to if I'm just running the tests to check if they pass.

Now that you mention it though, I do realise that `test:watch` is more descriptive. Until/unless we add more differences in addition to just `--watch` in the future, a better choice would be to just use `test:watch` :+1: 

Thanks!